internal/types/metadata_test.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestMaintainer(t *testing.T) {
+	assert.Check(t, is.Equal(Maintainer{"foo", "foo@bar.com"}.String(), "foo <foo@bar.com>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", ""}.String(), "foo <>"))
```

Comment:
This one should probably also *not* display the `<>` :stuck_out_tongue_closed_eyes: